docs/oauth.md

```diff
@@ -0,0 +1,73 @@
+# Testkube UI Authentication
+
+Testkube doesn't provide a separate user/role management system to protect access to its UI.
+Users can configure OAuth based authenication module using Testkube Helm chart parameters.
+Testkube can automatically create OAuth2-Proxy service and deployment integrated 
+with GitHub, as well as properly configure Kubernetes Nginx Ingress Controller and create required 
+ingresses.
+
+# Provide Parameters for UI and API Ingresses
+
+## API Ingress
+
+Pass values to Testkube Helm chart during installation or upgrade (they are empty by default).
+Pay attention to the usage of the scheme (http or https) in uris.
```

Comment:
Should this be "scheme" or "schema"?

Reply:
I beleive, it's scheme https://en.wikipedia.org/wiki/List_of_URI_schemes

Reply:
Got it.